bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -283,6 +283,14 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                concord::crypto::SignatureAlgorithm::EdDSA,
                "A flag to specify the operator message signing algorithm. It is defaulted to use EDDSA algo.");
 
+  CONFIG_PARAM(keysHistoryMaxBlocksNum,
+               std::uint64_t,
+               1024u,  // TODO (efrat): default value TBD
+               "threshold for pruning from keys history, versions below latest_block_id - this_value can be pruned."
+               "If set to 0, versions below the genesis block will be pruned");
```

Comment:
Can you add to the description:
"If set to 0, versions below the genesis block will be pruned, i.e. the whole available history will be kept (archival)."

Reply:
added thanks